Makefile

```diff
@@ -1,37 +1,13 @@
 all:
 
-PROG = bill
+compiler := src/compiler
```

Comment:
Independent definitions where each Makefile is allows for manual directory dependency.